datasets/gov/nvsc/uzkreciamos_ligos/sukelejai.csv

```diff
@@ -0,0 +1,68 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/nvsc/uzkreciamos_ligos/sukelejai,,,,,,,,,,,,Nacionalinio visuomenės sveikatos centro užkrečiamų ligų sukelėjų duomenys,
+,,,,,,,,,,,,,,
+,,,,Sukelejai,,,,,,3,,,Užkrečiamųjų ligų sukelėjai,
+,,,,,centras,string,,,,2,open,,NVSC departamenas,
+,,,,,diagnozes_kodas,ref,/datasets/gov/vlk/tlk10am/Tlk10am[code],,,4,open,,Diagnozės kodas,
+,,,,,diagnoze,string,,,,4,open,,Diagnozės pavadinimas,

```

Comment:
ar tai ne `ref` tipo laukas?
ar duomenys neateina iš to pačio rinkinio `/datasets/gov/vlk/tlk10am/Tlk10am`?